### ISSUE TITLE
VGG model predicts the same class always.

### ISSUE THREAD

**Issue by kabirahuja2431** [tanmoyopenroot/cancer_nn#5]
Hii.
I was also working on the same problem of skin cancer classification. While performing transfer learning with vgg16 my model always predict the same class for validation set i.e I have 150 images in my val set 120 of one category and 30 of other and I am always getting an accuracy of 0.8 on val set . Are you also facing the same issue?

Thanks

**Comment by tanmoyopenroot** [comment]
Balance the validation and training dataset because ( 120/ 150 ) gives you 0.8. The model is predicting 150 images of same category thereby giving you 0.8 accuracy. Also Increase the training dataset by augmenting.

**Comment by kabirahuja2431** [comment]
Thanks I will try that and get back to you :)

**Comment by kabirahuja2431** [comment]
Also, can you tell me the best validation accuracy that you got? Thanks

**Comment by vedadeepta** [comment]
Transfer learning on imagenet dataset(vgg16) gave us an accuracy of 67-68%. Further fine tuning the last block increases it to 69%. 
Edit: these accuracies are on validation data.

**Comment by kabirahuja2431** [comment]
Cool. Thanks a lot
